### ISSUE TITLE
[Feature Request] Compatibility List site.

### ISSUE THREAD

**Issue by nathanielcwm** [Arisotura/melonDS#497]
Maybe there could be a dedicated website like some other emulators where people can report game compatibility / check the compatibility of a game.

### Examples:
Dolphin: https://dolphin-emu.org/compat/
Citra: https://citra-emu.org/game/
Vita3K: https://vita3k.org/compatibility.html
Wine: https://appdb.winehq.org/
PCSX2: https://pcsx2.net/compatibility-list.html
TweakCompatible: https://jlippold.github.io/tweakCompatible/

**Comment by poudink** [comment]
yeah that would be very cool. there's some super outdated compatibility lists on the board but they're so old they're basically worthless.

**Comment by Arisotura** [comment]
thing is, I'd need some dedicated testers for filling it in (basically testing games in-depth, not just starting the first level and saying it's 'perfect')

I'm not going to have the time and patience to do it all myself, so this is why no such list exists

also, melonDS has gotten to a point where it runs almost anything you throw at it, so... yeah

**Comment by poudink** [comment]
it can be in a "wiki" format where anybody can modify it

**Comment by Arisotura** [comment]
sure, ala Dolphin

we'd also (mostly) need to have a page summarizing it and giving nice compatibility percentages and all

**Comment by MojoJojoDojo** [comment]
I think most of the NDS games that use all sorts of peripherals are gonna be emulated correctly in GBE+ later on.
It would be cool if they worked with melonDS tho.

**Comment by nathanielcwm** [comment]
maybe there could be a wiki page for each game listing known issues and fixes, peripherals etc
